Sources/AppKitNavigationShim/shim.m

```diff
@@ -0,0 +1,152 @@
+#if __has_include(<TargetConditionals.h>)
```

Comment:
This file can be deleted, too, I think, and brought back in the PR that introduces navigation.